### PR TITLE
SSB: Add Brandon

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -135,6 +135,48 @@ let BattleMovedex = {
...
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			// delta stream unable to be done without adding the weather for the move duration then removing it after
+			this.add('-anim', source, "Defog", target);
+		},
+		onHit: function (target, source, move) {
+			/**@type {?boolean | number} */
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:156]: Isn't used as a number though

Comment (Volco) [reply mods/seasonal/moves.js:156]: weirdly enough that's how defog's success variable is type scripted, i c/p'd the move to make it easier in this case. I changed it on my editor at the time of this comment and ran npm test, seems removing the `` | number`` creates no issue. I can remove it if you want.

Comment (HoeenCoder) [reply mods/seasonal/moves.js:156]: yeah just make it `/** @type {boolean} */`

**Comment by ghost** [comment]
@Volco conflix and bump

**Pull request closed without merging**